### ISSUE TITLE
Better instructions

### ISSUE THREAD

**Issue by ghost** [losuler/com.cisco.PacketTracer#3]
Thank you for this project, it worked on my Fedora 35 install (and I need this software for school), but it took me a good hour or so because I've never built a flatpak before.

Maybe include in the README the following commands:

flatpak-builder --install ./build ./com.cisco.PacketTracer.yml

flatpak install org.freedesktop.Platform/x86_64/20.08 org.freedesktop.Sdk/x86_64/20.08

This should be more clear to anyone hoping to install this. Alternatively, can this be put on something like flathub?

**Comment by losuler** [comment]
Thanks for bringing this up (I'm using Fedora too, using this software at university). When I wrote the docs for this I wanted to try something more generic and simple, forgetting I'm already fairly familiar with Flatpak. So I've added the commands for it now.

About having this on Flathub, as much as I want the same, access to the official `.deb` requires a Netacad account and Flatpak currently doesn't offer a way to provide account credentials on install for this. It would have to be a discussion to be had with the Flathub devs.
